rotkehlchen/tests/api/test_counterparty_asset_mappings.py

```diff
@@ -0,0 +1,279 @@
+        filter_query=CounterpartyAssetMappingsFilterQuery.make(offset=0, limit=5000),
+    )
+    return {
+        'entries': mappings,
+        'entries_found': mappings_found,
+        'entries_total': mappings_total,
+    }
+
+
+def test_counterparty_asset_mappings_query(
+        rotkehlchen_api_server: 'APIServer',
+        globaldb: 'GlobalDBHandler',
+) -> None:
+    # query all the mappings
+    result = _get_all_counterparty_mappings(globaldb)
+    assert len(result['entries']) == result['entries_found'] == result['entries_total'] == NUM_PACKAGED_COUNTERPARTY_ASSETS_MAPPINGS  # noqa: E501
+
+    # query all common mappings
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'counterpartyassetmappingsresource',
+        ),
+        json={'counterparty': None},
+    )
+    result = assert_proper_sync_response_with_result(response)
+    assert len(result['entries']) == result['entries_found'] == 0
+
+    # query all kraken mappings
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'counterpartyassetmappingsresource',
+        ),
+        json={'counterparty': 'hyperliquid'},
+    )
+    result = assert_proper_sync_response_with_result(response)
+    assert len(result['entries']) == result['entries_found'] == 0
+
+    # query by symbol all the kraken mappings
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'counterpartyassetmappingsresource',
+        ),
+        json={'counterparty_symbol': 'HYPE', 'counterparty': 'hyperliquid'},
+    )
+    result = assert_proper_sync_response_with_result(response)
+    assert all(entry['counterparty'] == 'hyperliquid' for entry in result['entries'])
+
+
+def test_counterparty_asset_mappings_add(
+        rotkehlchen_api_server: 'APIServer',
+        globaldb: 'GlobalDBHandler',
+) -> None:
+    all_mappings = _get_all_counterparty_mappings(globaldb)['entries']
+    added_mappings = [{
+        'asset': 'eip155:1/erc20:0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984',
+        'counterparty_symbol': 'NEW1',
+        'counterparty': 'hyperliquid',
+    }, {
+        'asset': 'eip155:1/erc20:0xF57e7e7C23978C3cAEC3C3548E3D615c346e79fF',
+        'counterparty_symbol': 'NEW2',
+        'counterparty': 'hyperliquid',
+    }]
+
+    # add two new mappings
+    response = requests.put(
+        api_url_for(
+            rotkehlchen_api_server,
+            'counterpartyassetmappingsresource',
+        ),
+        json={
+            'entries': added_mappings,
+        },
+    )
+    result = assert_proper_sync_response_with_result(response)
+    assert result is True
+
+    # check that the mappings were indeed added
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'counterpartyassetmappingsresource',
+        ),
+        json={
+            'offset': 0,
+            'limit': 4000,
+        },
```

Comment:
Don't explode things to take up multiple lines if not needed

```suggestion
        json={'offset': 0, 'limit': 4000},
```